themes.mdx

```diff
@@ -24,6 +28,8 @@ For example, to use the `maple` theme, you can add the following to your `docs.j
 
 There are a variety of themes to choose from.
 
+Each theme has its own unique set of component placements and styles.
```

Comment:
i feel like this could be misleading because we typically use the word "component" for the components listed [here](https://mintlify.com/docs/content/components/accordions) and they are shared across all themes.

Reply:
maybe the word "layout" is better?